### PR TITLE
Prac 05 code review request

### PR DESCRIPTION
**Pull request by callumjriley3**
Hi @talitha1234 - could you please check my use of code relating to sets and dictionaries in wimbledon.py, as well as variable names within each program? Much appreciated! :)

### CODE REVIEW

File: prac_05/wimbledon.py [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
 """
 
 FILENAME = "wimbledon.csv"
 
 
 def main():
+    """Process data from file and display champions, number of wins and winning countries."""
     champion_to_number_of_wins = {}
     countries = set()
     with open(FILENAME, "r", encoding="utf-8-sig") as in_file:
```
Comment (talitha1234) [inline prac_05/wimbledon.py:14]: Is this level of detail in main going to be too much for someone who doesn't know the program and is just looking to get an idea of what the program is doing overall?

File: prac_05/wimbledon.py [hunk 2]
```diff
@@ -1,13 +1,14 @@
...
-Actual:
+Actual: 42 minutes
 """
 
 FILENAME = "wimbledon.csv"
 
 
 def main():
+    """Process data from file and display champions, number of wins and winning countries."""
     champion_to_number_of_wins = {}
```
Comment (talitha1234) [inline prac_05/wimbledon.py:12]: Yes, dictionary use here makes sense to me.

File: prac_05/wimbledon.py [hunk 3]
```diff
@@ -1,13 +1,14 @@
...
     countries = set()
```
Comment (talitha1234) [inline prac_05/wimbledon.py:13]: Yes, also good choice using a set to get rid of any repeated country codes.

File: prac_05/word_occurrences.py [hunk 4]
```diff
@@ -1,7 +1,7 @@
 """
 Word Occurrences
 Estimate: 30 minutes
-Actual:
+Actual: 18 minutes
 """
 
 max_word_length = 0
```
Comment (talitha1234) [inline prac_05/word_occurrences.py:7]: Is using an abbreviated word like 'max' in line with the naming conventions required in this subject?

File: prac_05/hex_colours.py
Comment (talitha1234) [inline prac_05/hex_colours.py:1]: Could line 13 and 17 be changed/moved so as not to repeat code where possible?

File: prac_05/state_names.py [hunk 5]
```diff
@@ -11,11 +11,10 @@
 for pair in CODE_TO_NAME.items():
     print(f"{pair[0]:{MAX_SHORT_STATE_LENGTH}} is {pair[1]}")
```
Comment (talitha1234) [inline prac_05/state_names.py:12]: Does the abbreviated part in the variable MAX_SHORT_LENGTH follow this subjects variable naming conventions in the style guide?

Comment (talitha1234) [reply prac_05/state_names.py:12]: A separate note about MAX_SHORT_STATE_LENGTH

Could MAX_SHORT_STATE_LENGTH be renamed to be more consistent with the dictionary CODE_TO_NAME where they both refer to the keys in the dictionary?

File: prac_05/wimbledon.py (hunk 1 above)
Comment (callumjriley3) [reply prac_05/wimbledon.py:14]: Good point! I'll see if I can simplify it.

File: prac_05/word_occurrences.py (hunk 4 above)
Comment (callumjriley3) [reply prac_05/word_occurrences.py:7]: I'll change it to the extended word so it doesn't get confused with the built-in name. Thanks for pointing that out!

File: prac_05/hex_colours.py
Comment (callumjriley3) [reply prac_05/hex_colours.py:1]: Absolutely! I'll move the statement outside the try-except structure to follow the DRY principle. :)

File: prac_05/state_names.py (hunk 5 above)
Comment (callumjriley3) [reply prac_05/state_names.py:12]: Both great points! I will change the constant name to MAXIMUM_CODE_LENGTH, to a) avoid confusion involving the built-in name 'max', and b) have consistent naming between the constant and dictionary.

**Pull request merged**